jenkins_build.sh

```diff
@@ -0,0 +1,47 @@
+set -e
+
+REPO_NAME=zenoss-prodbin
```

Comment:
Please add checks for required variables which are empty/undefined. This helps people who are unfamiliar with the script catch operator errors more quickly.  For example, see https://github.com/zenoss/zenoss-deploy/blob/develop/jenkins/import-ova-ami.sh#L20